conf/default.yaml

```diff
@@ -72,3 +72,7 @@ plugins:
     jobs: 'DIRNAME/jobs'
     processEvery: '30 minutes'
     every: {}
+  hapi-metrics:
+    _nativePlugin: true
+    connectionUrl: 'MONGOURL'
+    collection: 'rapptor-metrics'
```

Comment:
Should we have an app name property that gets used to generate something like app-metrics?  Could be used for agenda too